examples/specs/layer_bar_dual_axis_minmax.vl.json

```diff
@@ -0,0 +1,64 @@
+{
```

Comment:
What's neat about this example is that it uses shared and independent scales in the same layered chart (with layer of layer) for the same channel.